tools/mo/openvino/tools/mo/ops/lstm_sequence.py

```diff
@@ -60,6 +60,11 @@ def supported_attrs(self):
     def backend_attrs(self):
         return [
```

Comment:
Also, do we need extender for this operation? 
Also, please check other RNNSequence operations?

Reply:
Added handling for GRUSequence operation.